docs/ff-concepts/design-system/design-system.md

```diff
@@ -333,9 +335,700 @@ You can modify its color, background color, and stroke width. In the 'Preview' s
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/oMAqsibJi8B9EdLgVanQ?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+7. Open your FlutterFlow project, navigate to the **Theme Settings** (from navigation menu) **> Typography & Icons**.
+8. Scroll down to the **Custom Icons** section.
+9. Click on the **+ Add Icons** button.
+10. Click on the **Upload Icon File** button.
+11. Select and upload `.ttf` file under the downloaded folder > fonts.
+12. Now click on the **Upload Icon Info** button.
+13. Select and upload the `filename.dart` under the downloaded folder (besides the fonts folder).
+14. Click **Add Icons**.
+15. To use a custom icon, add the **Icon** widget, move to the properties panel, and scroll down to the **Icon** section.
+16. Click on the already selected icon, select the **Custom Icons** tab, and then select your icon.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/DYKzHIQ27EaCiQArRlCT?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+If you prefer watching a video tutorial, here is the guide for you:
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.youtube.com/embed/rlGkbnhP75g"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Video guide
+
+If you prefer watching a video tutorial, here's the one for you:
```

Comment:
This video should be after the Custom Fonts section.

Reply:
Done!